### PR TITLE
[ID] Rework Welcome, Installation, and Registration

### PR DESCRIPTION
**Pull request by Nivalyx**
continuation of http://github.com/ppy/osu-wiki/pull/4166 c:

should hopefully be good to go as is as well but reviews are welcomed as always.

### CODE REVIEW

File: wiki/Installation/id.md [hunk 1]
```diff
@@ -2,76 +2,76 @@
...
 5. Terakhir, osu! akan meminta kamu untuk masuk atau mendaftar.
    - Jika kamu sudah memiliki akun, silakan masuk.
-   - Jika kamu perlu mendaftar, lihat artikel [Registrasi](/wiki/Registration), lalu masuk.
+   - Jika kamu belum memiliki akun, harap untuk terlebih dahulu mendaftar sesuai dengan panduan yang tertera pada artikel [Pendaftaran](/wiki/Registration).
 
-## Penyelesaian masalah
+## Penyelesaian masalah (troubleshooting)
 
-Jika kamu menggunakan Windows 7 64-bit, kamu mungkin akan mengalami beberapa kendala saat memasang osu!. Kamu bisa coba memasang .NET Framework versi 64-bit, jika program ini tidak disertakan dalam penginstal osu. [Unduh di sini](https://download.microsoft.com/download/2/0/e/20e90413-712f-438c-988e-fdaa79a8ac3d/dotnetfx35.exe)
+Aoabila kamu menggunakan Windows 7 64-bit, ada kemungkinan kamu akan dihadapkan dengan beberapa kendala teknis di dalam proses pemasangan osu!. Cobalah untuk memasang .NET Framework versi 64-bit secara terpisah terlebih dahulu apabila program ini tidak disertakan dalam installer osu. [Unduh di sini](https://download.microsoft.com/download/2/0/e/20e90413-712f-438c-988e-fdaa79a8ac3d/dotnetfx35.exe)
```
Comment (troke12) [inline wiki/Installation/id.md:26]:
```suggestion
Apabila kamu menggunakan Windows 7 64-bit, ada kemungkinan kamu akan dihadapkan dengan beberapa kendala teknis di dalam proses pemasangan osu!. Cobalah untuk memasang .NET Framework versi 64-bit secara terpisah terlebih dahulu apabila program ini tidak disertakan dalam installer osu. Silahkan [unduh di sini](https://download.microsoft.com/download/2/0/e/20e90413-712f-438c-988e-fdaa79a8ac3d/dotnetfx35.exe)
```

File: wiki/Installation/id.md [hunk 2]
```diff
@@ -2,76 +2,76 @@
...
+**Jika installer osu! terus-menerus menampilkan pesan error dalam proses pemasangannya, kamu dapat mencoba beberapa hal berikut:**
 
-- Coba pasang ulang .NET Framework dan pastikan untuk menggunakan versi terbaru. [Kamu dapat mengunduhnya lewat tautan berikut](https://dotnet.microsoft.com/download)
-- Bersihkan PC kamu - Pindai dengan anti-virus, bersihkan juga registry-nya. Program yang bagus untuk melakukan pembersihan umum adalah CCleaner. Pindai file komputer dan registry kamu. Penginstal osu! bisa saja gagal dijalankan dikarenakan ada entri registry osu! dari salinan osu! yang masih ada dalam sistem kamu.
+- Cobalah untuk memasang .NET Framework versi terbaru. [Kamu dapat mengunduhnya pada tautan berikut.](https://dotnet.microsoft.com/download)
+- Bersihkan PC kamu - Pindai dengan anti-virus, serta bersihkan juga registry-nya. Program yang disarankan untuk melakukan pembersihan umum adalah CCleaner. Pindai file komputer dan registry kamu. Installer osu! bisa saja gagal dijalankan dikarenakan ada entri registry osu! dari salinan osu! terdahulu yang masih belum terhapus sepenuhnya dalam sistem kamu.
```
Comment (troke12) [inline wiki/Installation/id.md:33]:
```suggestion
- Bersihkan PC kamu - Pindai dengan anti-virus, serta bersihkan juga registry-nya. Program yang disarankan untuk melakukan pembersihan adalah CCleaner. Pindai file komputer dan registry kamu. Installer osu! bisa saja gagal dijalankan dikarenakan ada entri registry osu! dari salinan osu! terdahulu yang masih belum terhapus sepenuhnya dalam sistem kamu.
```

File: wiki/Installation/id.md [hunk 3]
```diff
@@ -2,76 +2,76 @@
...
 
-## Menambahkan beatmapset
+## Menambahkan beatmap
 
 *Lihat juga: [Beatmaps](/wiki/Beatmaps)*
 
-Ada dua cara untuk menambahkan beatmap, melalui situs web atau osu!direct (memerlukan [osu!supporter](/wiki/osu!supporter)).
+Terdapat dua cara untuk menambahkan beatmap ke dalam klien permainan, yaitu dengan mengunduh secara manual melalui situs web osu! atau dengan menggunakan osu!direct (yang memerlukan [osu!supporter](/wiki/osu!supporter)).
```
Comment (troke12) [inline wiki/Installation/id.md:39]:
```suggestion
Terdapat dua cara untuk menambahkan beatmap ke dalam osu!, yaitu dengan mengunduh secara manual melalui situs web osu! atau dengan menggunakan osu!direct (yang memerlukan [osu!supporter](/wiki/osu!supporter)).
```
pake osu! biar lebih gampang dimengerti

**Comment by Nivalyx** [comment]
applied all suggestions!

**Review by troke12 — DISMISSED** [review]

**Comment by TicClick** [comment]
for the reference, the PR was restructured and merged in #4229

**Pull request closed without merging**